support-frontend/assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -84,4 +85,27 @@ export const tests: Tests = {
     seed: 3,
   },
 
+
+  digitalPackMonthlyOfferTest: {
+    type: 'OTHER',
+    variants: [
+      {
+        id: 'control',
+      },
+      {
+        id: 'one-for-one',
+      },
+    ],
+    audiences: {
+      ALL: {
+        offset: 0,
+        size: 1,
+      },
+    },
+    isActive: true,
+    referrerControlled: false,
+    seed: 3,
```

Comment:
Might want to seed this different to the above in case anyone decides to find their way to the contributions checkout and make a purchase 😆

Reply:
Oh it was different when I initially set it, they must have changed it in the meantime!